pkg/dogstatsd/listeners/udp.go

```diff
@@ -93,10 +94,13 @@ func NewUDPListener(packetOut chan Packets, sharedPacketPool *PacketPool) (*UDPL
 
 // Listen runs the intake loop. Should be called in its own goroutine
 func (l *UDPListener) Listen() {
+	var t1, t2 time.Time
 	log.Infof("dogstatsd-udp: starting to listen on %s", l.conn.LocalAddr())
 	for {
 		udpPackets.Add(1)
```

Comment:
unrelated to your changes, but this line should be moved to after the `ReadFrom` call, next to `tlmUDPPackets.Inc("ok")`, probably good to fix this while you're at it.